cmd/preflight/cli/run.go

```diff
@@ -129,6 +129,23 @@ func runPreflights(v *viper.Viper, arg string) error {
 		KubernetesRestConfig:   restConfig,
 	}
 
+	if v.GetString("since-time") != "" {
+		if v.GetString("since") != "" {
+			progressChan <- errors.Errorf("Only one of since-time / since may be used. The flag since-time will be used.")
+		}
+		for _, collectors := range preflightSpec.Spec.Collectors {
+			if collectors.Logs != nil {
+				collectors.Logs.Limits.SinceTime = v.GetString("since-time")
```

Comment:
`Limits` can be `nil` here. Might wanna initialize it before assigning the value to `SinceTime`. Same for other occurrences.

Reply:
Done!